src/components/header/Header.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import './_Header.scss'
+import imageLogo from '../../assets/compact-disc-solid.png'
 
 const Header = () => {
     return <header className="header">
-        <img src="" alt="logo" />
-        <p>We have over 2,000 songs, ready for you. Search, play and enjoy. Start typing and search through our catalog</p>
+        <div className="header__logo">
+            <img className="header__logo__icon" src={imageLogo} alt="logo streamify" />
```

Comment:
This is breaking the BEM naming convention.
http://getbem.com/naming/

You don't want to nest blocks. Create another block if necessary. So avoid doing:
`
 <img className="**header__logo__icon**" />
`
A block is a piece of the UI, and an element is a part of such block that wouldn't make sense on its own, so it could be something like:

`
<header className="header">
    <div className="header__container">
        <img className="header__logo" src={imageLogo} alt="logo streamify" />
          <span className="header__brand">Streamify</span>
    </div>
    <p className="header__legend">We have over 2,000 songs, ready for you. Search, play and enjoy. <br /> Start typing and search through our catalog</p>
</header>
`

Reply:
Attended